docs/how-it-works/internals.md

```diff
@@ -0,0 +1,120 @@
+# Internal Architecture
+
+> Last edited April 2024.
+
+This document will outline Lonboard's internal architecture, and hopefully should be useful for potential contributors.
+
+## Pydeck
+
+Before we consider Lonboard, we should briefly cover how Pydeck manages data transit.
+
+<!-- It's impossible to consider Lonboard's architecture in isolation because many of Lonboard's architectural decisions were chosen in explicit contrast to pydeck. Lonboard and pydeck both bind to deck.gl, but have significantly different APIs and characteristics. -->
```

Comment:
> both bind to deck.gl

I would actually argue that lonboard modifies the internals of deck layers to such an extent that it is more accurate to think of it as a Python GeoArrow rendering toolkit that is built on top of deck.gl, rather than a binding